### PR TITLE
DOCS/mpv: update mpv.conf's documentation

### PR DESCRIPTION
**Pull request by guidocella**

### CODE REVIEW

File: DOCS/man/mpv.rst [hunk 1]
```diff
@@ -629,30 +629,34 @@ user-specific one is ``~/.config/mpv/mpv.conf``. For details and platform
...
 User-specific options override system-wide options and options given on the
-command line override either. The syntax of the configuration files is
-``option=value``. Everything after a *#* is considered a comment. Options
-that work without values can be enabled by setting them to *yes* and disabled by
-setting them to *no*. Even suboptions can be specified in this way.
+command line override both. The syntax of the configuration files is
+``option=value``. Everything after a *#* is considered a comment. Options that
+work without values can be enabled by setting them to *yes* and disabled by
+setting them to *no*, and if the value is omitted, *yes* is implied. Even
+suboptions can be specified in this way.
```
Comment (Dudemanguy) [inline DOCS/man/mpv.rst:636]: Is this exactly the same or am I just missing something?

File: DOCS/man/mpv.rst [hunk 2]
```diff
@@ -666,7 +670,7 @@ Option                  Configuration file entry
 ``--flag``              ``flag``
 ``-opt val``            ``opt=val``
 ``--opt=val``           ``opt=val``
-``-opt "has spaces"``   ``opt="has spaces"``
+``-opt "has spaces"``   ``opt=has spaces``
```
Comment (Dudemanguy) [inline DOCS/man/mpv.rst:673]: Had no idea this even worked...

File: DOCS/man/mpv.rst (hunk 1 above)
Comment (Dudemanguy) [reply DOCS/man/mpv.rst:636]: Oh I can't read.

File: DOCS/man/mpv.rst (hunk 2 above)
Comment (guidocella) [reply DOCS/man/mpv.rst:673]: Me and avih had to reverse engineer the mpv.conf parsing code one year ago on IRC to discover how it really works, lol. It was high time we fixed the documentation. Actually another undocumented thing is that you can have `--` before option names in mpv.conf and it still works, but I don't think there is value in mentioning that.

**Pull request merged**